src/main/java/Duke.java

```diff
@@ -17,28 +17,42 @@ public static void main(String[] args) {
+            if (line.equals("bye")) {
                 break;
-            } else if(line.equals("list")){
+            } else if (line.equals("list")) {
                 System.out.println("____________________________________________________________");
                 System.out.println("Here are the tasks in your list:");
-                for(int j=0;j<count;j++) {
-                    System.out.println(j + 1 + "." + list[j].getStatusIcon() + list[j].description);
+                for (int j = 0; j < count; j++) {
+                    System.out.println(j + 1 + "." +list[j].toString());
                 }
                 System.out.println("____________________________________________________________");
-            } else if(line.contains("done")) {
+            } else if (line.contains("done")) {
                 String[] temp = line.split(" ");
                 int index = Integer.parseInt(temp[1]);
-                list[index-1].markAsDone();
+                list[index - 1].markAsDone();
                 System.out.println("Nice! I've marked this as done:");
-                System.out.println( list[index-1].getStatusIcon() + list[index-1].description);
-            } else {
-                list[i] = new Task(line);
-                count++;
+                System.out.println(list[index - 1].toString());
+            } else if(line.contains("todo")) {
+                list[count++] = new ToDo(line.substring(4));
                 System.out.println("____________________________________________________________");
-                System.out.println("Added: "+ line);
+                System.out.println("Got it. I've added this task:"+System.lineSeparator()+list[count-1].toString());
+                System.out.println("Now you have " + count + " tasks in your list.");
+                System.out.println("____________________________________________________________");
+            } else if(line.contains("deadline")){
+                String[] temp = line.split("/");
+                list[count++] = new Deadline(temp[0].substring(8),temp[1]);
+                System.out.println("____________________________________________________________");
+                System.out.println("Got it. I've added this task:" +System.lineSeparator()+list[count-1].toString());
+                System.out.println("Now you have " + count + " tasks in your list.");
+                System.out.println("____________________________________________________________");
+            } else if(line.contains("event")){
+                String[] temp = line.split("/");
+                list[count++] = new Event(temp[0].substring(5),temp[1]);
+                System.out.println("____________________________________________________________");
+                System.out.println("Got it. I've added this task:" +System.lineSeparator()+list[count-1].toString());
+                System.out.println("Now you have " + count + " tasks in your list.");
```

Comment:
The code seems a bit repetitive here. It would be great if you modify this part to be less repetitive. The lines here also seem to be a bit long.